Bio/PDB/ResidueDepth.py

```diff
@@ -292,7 +292,7 @@ def _get_atom_radius(atom, rtype="united"):
         return _atomic_radii[2][typekey]
     elif resname == "VAL" and at_name in {"CG1", "CG2"}:
         return _atomic_radii[9][typekey]
-    elif at_name in {"CD", "CD"}:
```

Comment:
This is clearly odd - and hints at a typo dating back to the first check-in of this code.